pkg/scheduler/objects/application.go

```diff
@@ -1147,13 +1140,13 @@ func (sa *Application) tryPreemption(reserve *reservation, ask *AllocationAsk) b
 	// Are there any victims/asks to preempt?
 	victims := preemptor.GetVictims()
 	if len(victims) > 0 {
+		log.Logger().Info("Found victims for daemon set ask preemption ",
```

Comment:
Given we've renamed this feature to 'required node preemption', can we change this message to reflect that? Maybe "Found victims for required node preemption "

Reply:
Done